data/data/openstack/masters/main.tf

```diff
@@ -132,6 +132,14 @@ resource "openstack_compute_instance_v2" "master_conf_0" {
     }
   }
 
+  dynamic "network" {
```

Comment:
Does terraform add these in the order they're defined? I hope so, and these need to be above openstack_additional_network_ids.

Reply:
No idea and OK